README.en.md

```diff
@@ -838,6 +838,47 @@ logging[path] = "/var/logs/error_log.log"
 logging[max_line_length] = 5124
 ```
 
+#### `translate_canonical_tag`
+Configures if WOVN.php should automatically translate existing canonical tag in the HTML. When set to `true`, WOVN.php
+will translate the canonical URL with the current language code. This setting defaults to `true`.
+
+Only language codes and aliases other than the default language will be included in the canonical URL. When the
+default language has an alias, it will be included in the canonical URL.
+
+`wovn.json`
+
+```json
+{
+  "translate_canonical_tag": true
+}
+```
+
+`wovn.ini`
+
+```ini
+translate_canonical_tag = true
+```
+
+#### `api_timeout_search_engine_bots`
+A variation of `api_timeout`, configures the translation API request timeout in seconds if WOVN.php has detected a request
+to be coming from a search engine bot. Currently, only GoogleBot is supported.
```

Comment:
the googlebot part needs updating

Reply:
Also we should add this documentation directly after `api_timeout` since they are related